components/sidebar/sidebar_category/sidebar_category_menu/sidebar_category_menu.tsx

```diff
@@ -126,13 +127,22 @@ class SidebarCategoryMenu extends React.PureComponent<Props, State> {
                 />
             );
         }
+        const createDirectMessage = (
```

Comment:
Do we want this to show up on every category menu or just the Direct Messages category?

Reply:
from what I understand from the ticket and spec it's all categories. @matthewbirtch can you confirm?

Reply:
I don't think there was any intention to add a direct message option to all category menus - just the DM category. Do you have a design reference where this is specified? I don't recall this recommendation.

There may be confusion. The intent was to add the option to create a new direct message from the top main `+` button, but not from every category.
![image](https://user-images.githubusercontent.com/2040554/101431265-83c6d400-38d4-11eb-9763-d696cae61800.png)

Reply:
@matthewbirtch there was a confusion but Harrison cleared it out for me. I added the button to the top menu as specified. You can review on the test server.

Reply:
Ok, thanks for clearing that up! Looks good to me